04037-chideraN.py

```diff
@@ -1 +1,2 @@
-print ('hello world,this is chidera NNAMADIM with HNGI7 ID HNG-04037 using python for stage 2 task bolunnamadim@gmail.com')
+name="chidera nnamadim" ID="HNG-04037" language="python" email="bolunnamadim@gmail.com"
+print("hello world,this is "+name+"with HNGi7ID"+ID+"using"+language+"for stage2task"+email)
```

Comment:
I can approve this but your script will not pass on the server. Please add whitespaces before and after your variables on the print line.